.gitignore

```diff
@@ -36,6 +36,7 @@ lib64/
 parts/
 sdist/
 var/
+yarn-v1.22.4/
```

Comment:
actually rather than something specific like this we should just ignore `dict/` instead (that should fix the same problem you were hitting)